internal/model/api/inflections.csv

```diff
@@ -1532,3 +1532,613 @@ Color:
+Eg:
+Km:
+Ni:
+Jm:
+Tk:
+Restricted:
+Bd:
+Bm:
+Et:
+Vg:
+Fj:
+Lu:
+Ve:
+Pm:
+Duns:
+Yyyy:
+Al:
+Jo:
+Pk:
+Ec:
+Ne:
+Pf:
+Ba:
+Uz:
+Co:
+Dj:
+Gd:
+Lr:
+Tv:
+Ar:
+Bl:
+Cl:
+Ge:
+Kw:
+Premium:
+Cz:
+Gh:
+It:
+La:
+Sk:
+Be:
+Cu:
+Kr:
+Za:
+Bb:
+Na:
+Vi:
+Se:
+Sl:
+Zw:
+Br:
+Mu:
+Pe:
+Sd:
+Ye:
+Dd:
+Ki:
+Kp:
+Ph:
+Kh:
+Ru:
+An:
+Er:
+Fk:
+Ptr:
+Central:
+Northeast:
+Cn:
+North:
+Https:HTTPS
+Soa:
+Txt:
+Spf:
+Aaaa:
+Upsert:
+Eu:
+Southeast:
+Str:
+Healthcheck:
+East:
+Tcp:TCP
+Srv:
+Insync:
+Hostedzone:
+West:
+Ap:
+Sa:
+Mx:
+Lb:
+Memcached:
+Ubiquitous:
+Rubygems:
+Php:PHP
+Rails:
+Bundler:
+Jvm:JVM
+Hvm:HVM
+Svn:SVN
+Java:
+Store:
+Haproxy:
+Stats:
+Ruby:
+Passenger:
+Nodejs:
+Ganglia:
+Dependencies:
+Paravirtual:
+Mysql:
+Nodejs:
+Aes:AES
+Horizon:
+Creating:
+Deleting:
+Updating:
+Resizing:
+Demand:
+Awaiting:
+Interrupted:
+Waiting:
+Ended:
+Core:
+Fulfillment:
+Bootstrapping:
+Arrested:
+Yellow:
+Debug:
+Warn:
+Updating:
+Launching:
+Green:
+Red:
+Grey:
+Fatal:
+Enabling:
+Disabling:
+Single:
+Projected:
+Ne:
+Le:
+Lt:
+Ge:
+Gt:
+Begins:
+Creating:
+Updating:
+Deleting:
+Eq:
+Between:
+Verifying:
+Deleting:
+Confirming:
+Finished:
+False:
+Eq:
+Le:
+Ge:
+Between:
+Eip:EIP
+Vpnconnection:VPNconnection
+Vpngateway:VPNgateway
+Ok:OK
+Apns:APNS
+Sandbox:
+Gcm:GCM
+Adm:ADM
+Tar:TAR
+Tgz:TGZ
+Autoscaling:
+Exclude:
+Queued:
+Fleet:
+Than:
+Milliseconds:
+Kilobytes:
+Terabytes:
+Greater:
+Microseconds:
+Gigabytes:
+Terabits:
+Less:
+Ok:OK
+Megabytes:
+Kilobits:
+Megabits:
+Gigabits:
+Equal:
+Simple:
+Structured:
+Lucene:
+Dismax:
+En:
+Hy:
+Mul:
+2xlarge:
+Light:
+Small:
+De:
+El:
+Fi:
+It:
+Ja:
+Ro:
+Th:
+Low:
+Ar:
+Hi:
+Ru:
+Fa:
+Ga:
+He:
+Nl:
+Tr:
+Xlarge:
+Bg:
+Zh:
+Latlon:
+Da:
+Fr:
+Gl:
+Lv:
+Sv:
+Hans:
+High:
+Eu:
+Pt:
+Hant:
+Minimal:
+Ca:
+Hu:
+Ko:
+Updating:
+Degraded:
+Production:
+Whitelist:
+Https:HTTPS
+Post:
+Sslv:SSLv
+Tlsv:TLSv
+Sni:SNI
+Vip:VIP
+Blacklist:
+Sni:SNI
+Vip:VIP
+Blacklist:
+Whitelist:
+Sslv:SSLv
+Tlsv:TLSv
+Https:HTTPS
+Post:
+Blacklist:
+Https:HTTPS
+Whitelist:
+Sni:SNI
+Vip:VIP
+Post:
+Cleanup:
+Do:
+Nothing:
+Detaching:
+Elbconnection:
+Proceed:
+Quarantined:
+Detached:
+Entering:
+Waiting:
+Pre:
+Mid:
+Workflows:
+Abandon:
+Unhandled:
+Children:
+Immediate:
+Mfaenabled:MFAenabled
+Assigned:
+Unassigned:
+Any:
+Csv:CSV
+Inprogress:
+Inactive:
+Mfadevices:MFAdevices
+Single:
+Enabling:
+Disabling:
+Ovm:
+Blackhole:
+Reachability:
+Performance:
+Hourly:
+Sold:
+Paravirtual:
+Retirement:
+Attaching:
+Machine:
+Cr:
+Retired:
+Cancelling:
+Bundling:
+Hvm:HVM
+Platforms:
+Ok:OK
+Applicable:
+Xlarge:
+4xlarge:
+One:
+Microsoft:
+Inactive:
+Heavy:
+Light:
+Deleting:
+2xlarge:
+Vhd:VHD
+Storing:
+Usd:
+Unix:
+Detached:
+Micro:
+Unixamazon:
+Gp:
+Xen:
+Detaching:
+Deregistered:
+Accepting:
+Windows:
+Small:
+Cg:
+Ipsec:IPsec
+Devpay:
+Ova:
+Store:
+Citrix:
+Stuck:
+Passed:
+Unresponsive:
+Creating:
+Linux:
+Io:
+Deny:
+Hi:
+Vmdk:VMDK
+Dedicated:
+Waiting:
+8xlarge:
+Vmware:VMware
+Utilization:
+Usd:
+Devpay:
+Ipsec:IPsec
+Io:
```

Comment:
It looks like Io is coming up twice, we probably want to make sure this list is uniq'd before adding the full thing.